src/System.Runtime/tests/System/IntPtrTests.cs

```diff
@@ -141,6 +141,7 @@ public static unsafe void ImplicitCast()
             Assert.Throws<OverflowException>(() => (int)ptr);
         }
 
+        [ActiveIssue(20154, TargetFrameworkMonikers.NetFramework)]
```

Comment:
disable for desktop -- this is by design, I see the bug was fixed in IntPtr.GetHashCode